### DISCUSSION TITLE
Multiple processes of coturn on 1 machine.

### DISCUSSION THREAD

**Discussion by WinstonPais** [coturn/coturn#1199] [Q&A]
I have a usecase where i want coturn to listen on
1) <port1> - UDP only by using `no-tls`
2) <port2> - TCP only by using `no-udp`

and want ssl enabled.

Currently, to achieve this I am running 2 processes of coturn.

Are there any issues that could arise due to this? 
For example any conflicts in the port range that gets assigned in candidate by coturn?

Also is it possible to achieve this in 1 process.
Noticed `alt-tls-listening-port=0` Is there any way i can define for which port i want these `no-tls` `no-udp` tags? 

Happy to create a PR if necessary.

**Comment by ggarber** [comment]
So you want coturn listening in UDP, TCP and TCP TLS but not UDP TLS ?
